views/update/review-beneficial-owner-individual.html

```diff
@@ -47,8 +47,27 @@ <h1 class="govuk-heading-xl govuk-!-margin-0">Review the individual beneficial o
       <input type="hidden" name="dob_year" value="{{ date_of_birth.year }}"/>
       <input type="hidden" name="dob_month" value="{{ date_of_birth.month }}"/>
 
-      <input type="hidden" name="date_of_birth" value="{{ date_of_birth }}"/>
+      {% if ch_reference %}
```

Comment:
What is this for?

Reply:
The whole address section I mean

Reply:
This is to populate the address fields after the change link. Although after realising that the ch_reference is set on get, this may need to change